hack/triage/config.yaml

```
@@ -0,0 +1,269 @@
+settings:
+  name: cluster-api
+  min_similarity: 0.8
+  repos:
+    - https://github.com/kubernetes-sigs/cluster-api
+  members:
+    - chrischdi
+    - enxebre
+    - fabriziopandini
+    - killianmuldoon
+    - sbueringer
+    - vincepri
+
+collections:
+  - id: initial-triage
+    name: Initial Triage
+    dedup: false
+    description: >
+      Check new issue and apply labels for kind, priority, area (optional).
+    rules:
+      - triage-needs-kind
+      - triage-needs-priority
+
+  - id: triage-refinement
+    name: Triage Refinement
+    dedup: false
+    description: >
+      Work on issue to get them to an actionable state.
+      When ready, apply triage/accepted and help of good-first issue label
+    rules:
+      - triage-stale-awaiting-more-evidence
+      - triage-stale-kind-support
+      - triage-updated
+      - triage-again
+      - triage-lifecycle-stale-or-rotten-priority-critical-urgent
+      - triage-lifecycle-stale-or-rotten-priority-important-soon
+      - triage-lifecycle-stale-or-rotten-priority-important-longterm
+      - triage-lifecycle-frozen
+      - triage-all
+
+  - id: actionable
+    name: Actionable
+    description: >
+      Issue to be worked.
+    dedup: false
+    rules:
+      - actionable-not-assigned-without-help-or-good-first-issue
+      - actionable-updated
+      - actionable-all
+
+  - id: fix-me
+    name: Fix-me!
+    description: >
+      Ensure we are using the labels in a consistent way.
+    dedup: false
+    rules:
+      - fixme-avoid-triage-needs-information
+      - fixme-avoid-kind-design
+      - fixme-kind-proposal-without-kind-feature
+      - fixme-kind-support-with-other-kinds
+      - fixme-triage-accepted-with-kind-support-or-priority-awaiting-more-evidence
+      - fixme-help-or-good-first-issue-before-triage-accepted
+      - fixme-lifecycle-frozen-with-priority-critical-urgent-priority-important-soon-priority-important-longterm
+
+rules:
+  # Phase: Initial triage
+
+  triage-needs-kind:
+    # WHY?: kinds need to be assigned by maintainers, let's do it!
+    name: "needs-kind"
+    resolution: "Add a kind/ label: api-change, bug, cleanup, deprecation, design, documentation, failing-test, feature, flake, proposal, regression, release-blocking, support"
```

```suggestion
    resolution: "Add a kind/ label: api-change, bug, cleanup, deprecation, documentation, failing-test, feature, flake, proposal, regression, release-blocking, support"
```
below we have a rule to not use kind/design